chainer/links/connection/gru.py

```diff
@@ -185,21 +185,20 @@ def __call__(self, x):
 class GRU(StatefulGRU):
     """Stateful Gated Recurrent Unit function (GRU)
 
-    This is an alias of "~chainer.links.StatefulGRU".
-    Its documented API is identical to the function.
+    This is an alias of :class:`~chainer.links.StatefulGRU`.
 
     .. warning::
 
        In Chainer v1, :class:`~chainer.links.GRU` was *stateless*,
```

Comment:
I think ``` :class:`~chainer.links.GRU` ``` does not have to be a link, because it's only about an entity in v1.
How about  ``` ``chainer.links.GRU`` ``` ?
Maybe without module prefix, i.e. ``` ``GRU`` ``` (to align with the link to StatefulGRU)?